fee_allocator/accounting/chains.py

```diff
@@ -43,7 +43,8 @@ def __init__(
         cache_dir: Path = None,
         use_cache: bool = True,
     ):
-        self.input_fees = input_fees
+        # convert wei fees to usd. identified by the lack of a decimal point
+        self.input_fees = {chain: fee / 1e6 if isinstance(fee, int) else fee for chain, fee in input_fees.items()}
```

Comment:
I get a little confused here with the use of the input_fee class, but I think the logic looks ok.  It seems like it would make more sense to run in native wei at the end, and it looks to me like we are translating everything to float, but thjat's a bigger change.